flexbox/cell.go

```diff
@@ -64,10 +64,10 @@ func (r *Cell) SetMinWidth(value int) *Cell {
 	return r
 }
 
-// SetMinHeigth sets the cells minimum height, this will not disable responsivness.
+// SetMinHeight sets the cells minimum height, this will not disable responsivness.
 // This has only an effect to cells of a HorizontalFlexBox.
```

Comment:
You can do this to avoid a breaking change

People will have warning in their IDE

Then at some point you could decide to remove SetMinHeigth

```suggestion
// Deprecated: use [*Cell.SetMinHeight]
 func (r *Cell) SetMinHeigth(value int) *Cell { 
          return r.SetMinHeight(value)
  }

// SetMinHeight sets the cells minimum height, this will not disable responsivness.
 // This has only an effect to cells of a HorizontalFlexBox.
```